components/function-controller/cmd/manager/main.go

```diff
@@ -66,15 +66,19 @@ func main() {
 		setupLog.Error(err, "unable to load config")
 		os.Exit(1)
 	}
-	l, err := serverlessLogging.ConfigureLogger(config.LogLevel, config.LogFormat)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	l, err := configureLogger(ctx, config.FileConfigPath)
```

Comment:
Why `l`? When I read `l.CreateNamed("controllers.role")` I now that this create named something but I have no idea what is created.
I now that `l` was in previous version of code, but it was used in short part of code. Now it is used is scope longer than 100 rows.

Reply:
good idea, I will change this!